HeartbeatLogging/Sources/Storage.swift

```diff
@@ -46,11 +46,12 @@ final class FileStorage: Storage {
 
```

Comment:
Should Storage have a `remove` function that still calls `removeItem`?

Reply:
There is no need for it now. The main API requirements are to:
-  read 
-  write data
-  empty file(`write(nil)`)